server/github.go

```diff
@@ -256,18 +263,19 @@ func (s *Server) areChecksSuccessfulForPr(pr *model.PullRequest, org string) (bo
 		return false, err
 	}
 	mlog.Debug("Retrieved status for pr", mlog.String("status", cStatus.GetState()), mlog.Int("prNumber", pr.Number), mlog.String("prSha", pr.Sha))
-	if cStatus.GetState() == stateSuccess || cStatus.GetState() == "pending" || cStatus.GetState() == "" {
+	if cStatus.GetState() == stateSuccess || cStatus.GetState() == statePending || cStatus.GetState() == "" {
 		return true, nil
 	}
 	return false, nil
 }
 
-func (s *Server) createRepoStatus(ctx context.Context, pr *model.PullRequest, status *github.RepoStatus) {
+func (s *Server) createRepoStatus(ctx context.Context, pr *model.PullRequest, status *github.RepoStatus) error {
```

Comment:
1/5 `createRepoStatus` should not return an error and be just fire and forget. Because the caller doesn't care most of the time if it succeeded. Logging to Mattermost can happen here

Reply:
I would argue that sometimes I do care very much (ee tests and CLA) and I don't want to make too much noise to mattermost when I do not care.